examples/others/lmcache/hpu/README.md

```diff
@@ -0,0 +1,74 @@
+# LMCache Examples
+Please Note: HPU integration for LMCache will be upstreamed. After that, the following test cases can be used.
+
+This folder demonstrates how to use LMCache for disaggregated prefilling, CPU offloading and KV cache sharing.
+
+## 1. Disaggregated Prefill in vLLM v1
+
+This example demonstrates how to run LMCache with disaggregated prefill using lm or redis on a single node.
+
+### Prerequisites
+- HPU implementation to be upstreamed
+- At least 2 HPUs
+- Valid Hugging Face token (HF_TOKEN) for Llama 3.1 8B Instruct.
+
+### Usage
+
+Run
+`cd disagg_prefill_lmcache_v1`
+to get into `disagg_prefill_lmcache_v1` folder, and then run
+
+```bash
+PT_HPU_GPU_MIGRATION=1 VLLM_USE_V1=1 VLLM_SKIP_WARMUP=True PT_HPU_ENABLE_LAZY_COLLECTIVES=true bash disagg_example.sh
+```
+
+to run disaggregated prefill and benchmark the performance.
+
+lmserver is default and it's configurable as well as tensor_parallel_size and model name.
+
+Example) redis server, tensor_parallel_size 4 and Llama-3.1-70B-Instruct model
+
+```
+PT_HPU_GPU_MIGRATION=1 VLLM_USE_V1=1 VLLM_SKIP_WARMUP=True PT_HPU_ENABLE_LAZY_COLLECTIVES=true bash disagg_example.sh -s redis -t 4 -m meta-llama/Llama-3.1-70B-Instruct
```

Comment:
is it necessary to use PT_HPU_GPU_MIGRATION=1?
Will have bring performance impact?